### PR TITLE
Adding Golgi first donation

### PR DESCRIPTION
**Pull request by GDiDieco**
Adding Golgi first donation

### CODE REVIEW

File: GOLGI/donations.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  recipient: 
+    name: Centre for Regenerative Medicine “Stefano Ferrari”
+    address: Via Glauco Gottardi 10 41125 Modena (Italy)
+    tel: +39.059.205811
+    contact: centromedicinarigenerativ@unimore.it
+
+  ada: 180 # <- These must be numbers. Can be decimals though. See below
+  usd: 212.5 # Only usd currency accepted, check on xe.com for exchange rate.
+  timestamp: 1599026880
```
Comment (ResponsibleStaking) [inline GOLGI/donations.yml:11]: This timestamp is more than 1 year ago. please use https://www.unixtimestamp.com/ to generate a matching timestamp.

**Review by ResponsibleStaking — CHANGES_REQUESTED** [review]

File: GOLGI/donations.yml (hunk 1 above)
Comment (GDiDieco) [reply GOLGI/donations.yml]: Hello, I have update timestamp on the repository.

**Comment by GDiDieco** [comment]
Hello,
thanks for your reply, I have updated timestamp with correct information.

Do you need other correction or other action from my side?

Thanks.

G.

Il 18/07/2021 23:37, ResponsibleStaking ha scritto:

**Review by ResponsibleStaking — APPROVED** [review]
Looks good now! Thanks for updating it!

File: GOLGI/donations.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  timestamp: 1625468400 
+  # ^ This is a unix timestamp of the moment you donated.
+  #   You can go to https://www.unixtimestamp.com/ to calculate your donation.
+
+  files:
+  # ^ a list of files relating to the donation. Usualy photos or screenshots 
+  #   of the receipts.
+  #   Make sure to add only relative paths that begin from the root of the 
+  #   repo (they should include your Ticker name).
+    - GOLGI/20210705_first donation Golgi Stake Pool.jpg
```
Comment (ResponsibleStaking) [inline GOLGI/donations.yml:20]: Sorry to bother you again. It seems the GitHub checks in place are not working with spaces in files names.
Could you please rename this file and the reference here. 
Unfortunately, I cannot approve as long as the automated checks are not passing.
Sorry for the inconvenience.

**Review by ResponsibleStaking — CHANGES_REQUESTED** [review]

**Comment by GDiDieco** [comment]
Hello,
no problem I uploaded new file with no spaces in the name and I have 
modified the .yml according to new filename..

I hope I understand correctly your advice.

Thanks.

G.

Il 19/07/2021 08:43, ResponsibleStaking ha scritto:

File: GOLGI/donations.yml (hunk 2 above)
Comment (GDiDieco) [reply GOLGI/donations.yml]: Hello,
no problem I uploaded new file with no spaces in the name and I have modified the .yml according to new filename..

I hope I understand correctly your advice.

Thanks.

**Review by ResponsibleStaking — APPROVED** [review]
I removed the comments. For some reason those made the availability check not to work.
Not everything is fine and the build was running through. 
Thanks for the prompt update!

**Comment by GDiDieco** [comment]
Perfect, thanks a lot for your avaliability.

See you soon.

G.

Il 19/07/2021 09:00, ResponsibleStaking ha scritto:

**Pull request merged**